src/components/Calculator.js

```diff
@@ -1,60 +1,54 @@
-import React from 'react';
+import { useState } from 'react';
 import calculate from '../logic/calculate';
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206870330-6422197d-d098-42bb-bb18-d8d59cd224af.PNG)

- [ ] When i launched your application and clicked on number 8 then i saw **undefined** on your screen 😨 which you can see in the image above 🤔 Kindly fix it 👍

Reply:
Hi @HAMMAS-SALEEM 
Thanks for pointing that out. Any tips on how I can resolve that?